framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -267,9 +257,10 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
     */
 
     /**
-     * Returns the boundaries of the pane.
+     * Returns the dimensions of the pane.
      *
-     * @return {Map} The pane boundaries.
+     * @return {{width: number, height: number}|null} The pane dimension in pixel. Contains
```

Comment:
@mever did you build the api viewer with your PR? I not: please do so and see if the api viewer shows this as expected, containing your changes. It it is OK leave it in. If not, please revert the change. Thank you.